src/movies.js

```diff
@@ -69,7 +69,37 @@ function orderByYear(a, b) {
 //console.log(movies);
 
 // Iteration 6: Alphabetic Order - Order by title and print the first 20 titles
-function orderAlphabetically() {}
+
+
+function orderAlphabetically(arr) {
+
+  const sortedMovies = movies.sort(function (a, b) {
+    if (a.title < b.title) {
+      return -1;
+    }
+    if (a.title > b.title) {
+      return 1;
+    }
+    return 0;
+  });
+
+console.log(sortedMovies)
+
+  const arrLength = sortedMovies.length;
+
+console.log(arrLength)
+
+  if (arrLength <= 3) {
+    return sortedMovies;
+  } else {
+    for (let i = 0; i > sortedMovies[3]; i++) {
+     return sortedMovies.pop();
```

Comment:
I am not entirely sure what you are trying to do here, what does  '<= 3' mean ? :)